14-location.Rmd

```diff
@@ -366,8 +352,7 @@ The result of the subsequent code chunk is therefore an estimate of shop density
 ```{r 14-location-25, message=FALSE, warning=FALSE}
 shops = sf::st_transform(shops, st_crs(reclass))
 # create poi raster
-poi = terra::rasterize(x = terra::vect(shops),
-                       y = reclass, field = "osm_id", fun = "length")
+poi = terra::rasterize(x = shops, y = reclass, field = "osm_id", fun = "length")
```

Comment:
are you sure we don't need to convert an sf-object into an vect-object? Ok, let's give it a try and see if it works -:)

Reply:
excellent, you are right, it works :+1:

Reply:
👍🏻